tests/spread/integration/bind9-dnsutils/task.yaml

```
@@ -0,0 +1,10 @@
+summary: Integration tests for bind9-dnsutils
+
+execute: |
+  rootfs="$(install-slices bind9-dnsutils_bins)"
+
+  chroot "${rootfs}/" delv -v
+  chroot "${rootfs}/" dig -v
+  chroot "${rootfs}/" mdig -v
+  chroot "${rootfs}/" nslookup -version
```

```suggestion
  chroot "${rootfs}/" delv @8.8.8.8 | grep -P '[a-m].root-servers.net'
  chroot "${rootfs}/" dig @8.8.8.8 example.com | grep ';; ANSWER SECTION:'
  chroot "${rootfs}/" mdig @8.8.8.8 example.com | grep ';; ANSWER SECTION:'
  chroot "${rootfs}/" nslookup example.com 8.8.8.8 | grep 'Name:[[:space:]]example.com'
```

Let's use some real-world test cases.

---

HI @zhijie-yang,

The above suggested changes fail in the Spread tests on the ARM64 host, from appear to be networking issues.
How would you suggest we proceed?
```
+ chroot /tmp/tmp.AJujICSL75/ delv @8.8.8.8
+ grep -P '[a-m].root-servers.net'
;; connection refused resolving './NS/IN': 8.8.8.8#53
;; resolution failed: SERVFAIL
-----
```

Thanks,
Jeremy

---

Hey, I'm still investigating and working on this. Will come back to you once some solutions are found.